resources/nl-core/nl-core-NameInformation.xml

```diff
@@ -35,7 +33,100 @@
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.4" />
+        <comment value="FirstNames (implicit, main mapping is on .given)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.5" />
+        <comment value="Initials (implicit, main mapping is on .given)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.6" />
+        <comment value="GivenName (implicit, main mapping is on .given in a sibling element)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.7" />
+        <comment value="NameUsage (implicit, main mapping is on extension:nameUsage)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.11" />
+        <comment value="Prefix (implicit, main mapping is on .family.extension:prefix)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.10" />
+        <comment value="LastName (implicit, main mapping is on .family.extension:lastName)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.9" />
+        <comment value="PartnerPrefix (implicit, main mapping is on .family.extension:partnerPrefix)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.8" />
+        <comment value="PartnerLastName (implicit, main mapping is on .family.extension:partnerLastName)" />
+      </mapping>
+    </element>
+    <element id="HumanName.family">
+      <path value="HumanName.family" />
+      <comment value="When implementing zib NameInformation, this rendition should contain a rendering of the Prefix, LastName, PartnerPrefix and PartnerLastName concepts, constructed according to the NameUsage concept." />
+      <condition value="nl-core-NameInformation-01" />
+      <constraint>
+        <key value="nl-core-NameInformation-02" />
+        <severity value="warning" />
+        <human value="If parts of the family name are specified using the extensions in .family, the .family element SHOULD be populated with the full family name as well. This ensures compatibility with common FHIR usage." />
+        <expression value="(extension('http://hl7.org/fhir/StructureDefinition/humanname-own-prefix').exists() or extension('http://hl7.org/fhir/StructureDefinition/humanname-own-name').exists() or extension('http://hl7.org/fhir/StructureDefinition/humanname-partner-prefix').exists() or extension('http://hl7.org/fhir/StructureDefinition/humanname-partner-name').exists()) implies $this.hasValue()" />

```

Comment:
Nice! Tested it with (name.family.extension('http://hl7.org/fhir/StructureDefinition/humanname-own-prefix').exists() or name.family.extension('http://hl7.org/fhir/StructureDefinition/humanname-own-name').exists() or name.family.extension('http://hl7.org/fhir/StructureDefinition/humanname-partner-prefix').exists() or name.family.extension('http://hl7.org/fhir/StructureDefinition/humanname-partner-name').exists()) implies name.family.hasValue() in Simplifier FHIRPath playground.